qcengine/programs/tests/test_molpro.py

```diff
@@ -27,20 +28,38 @@ def test_molpro_output_parser(test_case):
     output_ref = qcel.models.Result.parse_raw(data["output.json"]).dict()
     output_ref.pop("provenance", None)
 
-    # TODO add `skip` to compare_recusive
+    # TODO add `skip` to compare_recursive
     check = compare_recursive(output_ref, output)
     assert check, check
 
 
 @pytest.mark.parametrize('test_case', molpro_info.list_test_cases())
 def test_molpro_input_formatter(test_case):
 
-    # Get output file data
+    # Get input file data
     data = molpro_info.get_test_data(test_case)
     inp = qcel.models.ResultInput.parse_raw(data["input.json"])
 
-    # Just test that it runs for now
-    # TODO add actual comparison
+    # TODO add actual comparison of generated input file
     input_file = qcng.get_program('molpro').build_input(inp, qcng.get_config())
-    #print(input_file['infiles']['dispatch.mol'])
-    assert input_file.keys() >= {"commands", "infiles"}
\ No newline at end of file
+    assert input_file.keys() >= {"commands", "infiles"}
+
+
+@using_molpro
+@pytest.mark.parametrize('test_case', molpro_info.list_test_cases())
+def test_molpro_executor(test_case):
+    # Get input file data
+    data = molpro_info.get_test_data(test_case)
+    inp = qcel.models.ResultInput.parse_raw(data["input.json"])
+
+    # Run Molpro
+    result = qcng.get_program('molpro').compute(inp, qcng.get_config(local_options={"ncores": 4}))
+    assert result.success is True
+
+    # Get output file data
+    output_ref = qcel.models.Result.parse_raw(data["output.json"])
+
+    if result.success:
```

Comment:
The assert above will not let the computation get to this point.

Reply:
The test for Terachem also has this redundancy. Should I change it there too?

Reply:
Yes please!

Reply:
Do you mind if I sneak the update to Terachem in this PR? Or should I make a separate PR?

Reply:
Fine to add it here, imo.

Reply:
I have added the changes